test/suite/test_import11.py

```diff
@@ -159,6 +159,18 @@ def test_file_import(self):
         self.copy_files(self.bucket, os.path.join(newdir, self.bucket))
         self.copy_files(self.cache_bucket, os.path.join(newdir, self.cache_bucket))
 
+        # The file_metadata configuration should not be allowed in the tiered storage scenario.
+        if self.is_tiered_scenario():
+            invalid_import_config1 = 'import=(enabled,repair=false,file_metadata="")'
+            invalid_import_config2 = 'import=(enabled,repair=false,file_metadata="",metadata_file="WiredTiger.export")'
+            invalid_import_config3 = 'import=(enabled,repair=false)'
+        
+            err_msg1 = "/Invalid argument/"
```

Comment:
Do we get this message because `file_metadata` is empty and there's no `WiredTiger.export` file present on the disk?

Reply:
Sorry, I realised I mixed up the error messages. The current test doesn't run... So for `invalid_import_config3` it should return "invalid argument" message since this is the error you get: `import requires that the table configuration is specified or the 'repair' option is provided: Invalid argument`, however for `invalid_import_config1` and `invalid_import_config2` we should get the `import for tiered storage is incompatible with the 'file_metadata' setting` message.

Reply:
I can definitely experiment more with what happens if we try to provide a valid `file_metadata` value.

Reply:
Also, I do think that there is a valid `WiredTiger.export` file present, it looks like the lines of code afterwards are importing from this file.

Reply:
@AlexeyAnisimovAU I've added a value to the `file_metadata` config now.